cookbook/multi-server-setup.rst

```diff
@@ -0,0 +1,67 @@
+Running Sulu in a Multi-Server Setup
+=====================================
+
+Sulu is built with horizontal scaling in mind and allows to distribute the load of your application between multiple servers.
+In such a setup, it is important to configure your application to use centralized solutions for session management, cache management,
+search indexes and media storage.
+
+
+Media Storage
+-------------
+
+Media files that are uploaded in the media section of the Sulu administration interface are stored in the filesystem of the
+current server by default. If your application runs on multiple servers, you need to :doc:`configure an external storage adapter<./external-media-storage>`
+for your media files.
+
+
+HTTP Cache
+----------
+
+Sulu utilizes the `FOSHttpCacheBundle`_ to improve response times by caching rendered pages. This cache is automatically invalidated
+when the content of a page is changed. To prevent different cache entries on different servers, you need to :doc:`configure Varnish as a centralized caching proxy<./caching-with-varnish>`.
+
+
+Application Cache
+-----------------
+
+The Symfony cache improves the speed of your application by caching metadata, doctrine results and PHPCR data.
+If your application runs on multiple servers, you need to `configure a centralized caching adapter`_ like ``redis`` for your ``app`` cache.
+Additionally, if the application runs in different directories on different servers, you need to set a static ``prefix_seed``:
+
+.. code-block:: yaml
+
+    # config/packages/cache.yaml
+    framework:
+        cache:
+            app: cache.adapter.redis
+            prefix_seed: your_vendor_name/app_name
+            default_redis_provider: "redis://127.0.0.1:6379"
```

Comment:
would use a `ENV` here like this: https://github.com/sulu/sulu-demo/blob/80b6d2d31412b15e9dbefa0dd7182497f8539547/config/packages/cache.yaml#L15

https://github.com/sulu/sulu-demo/blob/80b6d2d31412b15e9dbefa0dd7182497f8539547/config/packages/cache.yaml#L1-L2

Reply:
I am not sure about that, I would prefer to keep the example as simple as possible. Otherwise, some people might think they are required to use environment variables and get confused how to configure them. 
If somebody wants to use environment variables, he probably knows that he can just replace the hardcoded value 🙂